CMakeLists.txt

```diff
@@ -84,6 +84,7 @@ openpmd_option(HDF5           "HDF5 backend (.h5 files)"                  AUTO)
 openpmd_option(ADIOS1         "ADIOS1 backend (.bp files)"                AUTO)
 openpmd_option(ADIOS2         "ADIOS2 backend (.bp files)"                AUTO)
 openpmd_option(PYTHON         "Enable Python bindings"                    AUTO)
+openpmd_option(CUDA           "Use CUDA"                                  AUTO)
```

Comment:
This is great, but only needed for tests/downstream (does not influence the API's feature set).

Let us name this `CUDA_TESTS` and set it to `OFF` by default, to avoid complications of building the API defaults (e.g., if CUDA is found but its not working properly). We can set it to ON explicitly for the workflows that we run on GPUs.

Reply:
OK, changed